mitxpro/templates/footer.html

```diff
@@ -38,7 +38,7 @@ <h2>Contact Us</h2>
             <div class="right-info">
                 <ul class="footer-nav">
                     <li><a href="/about-us/">About Us</a></li>
-                    <li><a href="/blog/">Blog</a></li>
+                    <li><a href="https://curve.mit.edu/">Blog</a></li>
                 </ul>
                 <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2-legacy.js"></script>
                 <![endif]-->
```

Comment:
I know this is not related to your code, but please remove this unused commented code as well.